go/libkb/sig_chain_test.go

```diff
@@ -342,15 +340,11 @@ func storeAndLoad(t *testing.T, tc TestContext, chain *SigChain) {
 		t.Fatal(err)
 	}
 
-	// Loading a chain from cache won't load all the links, if there's a
-	// sigchain reset in the middle. So we don't want to check the entire
-	// loaded chain for equality with what we saved. But we can make sure it's
-	// loaded all the links of the *current subchain*. HOWEVER: Loading
```

Comment:
The first half of this comment is no longer true, and we can now test the whole chain.